tests/cmake/hwm/board_extend/oot_root/boards/arm/mps2/mps2_an521-common.dtsi

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+sysclk: system-clock {
+	compatible = "fixed-clock";
+	clock-frequency = <25000000>;
+	#clock-cells = <0>;
+};
+
+timer0: timer@0 {
+	compatible = "arm,cmsdk-timer";
+	reg = <0x0 0x1000>;
+	interrupts = <3 3>;
```

Comment:
Wouldn't it be better to include the original board DTS, instead of copying it into the test?

Reply:
a copy was chosen as we just need a device tree where we can verify that it's being correctly extended.
We have no intention of tracking the real board, and only reason to have the real board in this case is to ensure the extension mechanism works.

Reply:
> We have no intention of tracking the real board

If that's the case, have you considered using something closer to blank/dummy files? I don't know if the dts contents here are essential to pass these tests.

Reply:
I did, but this is where I took the short route of a copy and instead focus on the board extend functionality itself.

Having the dts as close to the board it extends is probably more likely than an blank / dummy dts file.
Although you can say that for a real extend, a user would most likely try to include a dtsi file if possible, which is close to your first comment 😉